client/src/App.js

```diff
@@ -10,6 +13,15 @@ function App() {
         <Route exact path="/">
           <HomePage />
         </Route>
+        <Route exact path="/search">
```

Comment:
i don't think you need the "exact for the other paths :)

Reply:
You're right! Copy-pasted without thinking 🙈